webapp/src/pages/Settings.tsx

```diff
@@ -550,7 +601,8 @@ const Settings: React.FC = () => {
       {getAnalysesCustomization("syntax")}
       {displayToggleSectionTitle("similarity", "Similarity")}
       {getAnalysesCustomization("similarity")}
-      {displayToggleSectionTitle("behavioral_testing", "Perturbation Testing")}
+      {displayToggleSectionTitle("behavioral_testing", "Behavioral Testing")}
+      {displayBehavioralTestingSection()}
```

Comment:
Should we hide the section if the "Behavioral Testing" checkbox is not unchecked? I find it confusing that it stays. It could also be grayed out, but it's simpler to hide. What do you think/prefer?
```suggestion
      {resultingConfig.behavioral_testing && displayBehavioralTestingSection()}
```